spec/shr_allergy_map.txt

```diff
@@ -37,14 +37,11 @@ NoAdverseSensitivityToSubstanceStatement maps to http://hl7.org/fhir/us/core/Str
 	constrain onset[x] to 0..0
 	constrain lastOccurrence to 0..0
 	constrain reaction to 0..0
-
 		
-NoKnownAllergyStatement maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
+NoKnownAllergy maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
```

Comment:
This class is commented out in shr.allergy

Reply:
should be okay.  This is part of the comment block in shr_allergy_map.txt.

Reply:
Ah! Missed that. Thanks